Automattic/wp-calypso
# BBE: add images for the store flow

Fixes https://github.com/Automattic/martech/issues/2476

## Proposed Changes

* Adds images to the site build showcase for the store flow.

## Testing Instructions



* Go to `/start/do-it-for-me-store`.
* Confirm that the images in the slideshow match the one linked here: https://github.com/Automattic/martech/issues/2476#issue-2028040708

## Pre-merge Checklist
